tides-server/pkg/handler/user_api.go

```diff
@@ -10,7 +12,45 @@ import (
 
 	"tides-server/pkg/config"
 	"tides-server/pkg/models"
+
+	"crypto/tls"
+	gomail "gopkg.in/mail.v2"
+	"github.com/sethvargo/go-password/password"
 )
+const OFFICIAL_EMAIL = "cloudtest2021abc@gmail.com"
+const OFFICIAL_PASSWORD = "cloudtides"
```

Comment:
it is not recommended to hard coded any password in the code, if this email and password is very important, please remove this part and try to use environment variable.

Reply:
@yvvym for resolve